src/marine/hgodas_insitu2ioda.py

```diff
@@ -19,13 +19,14 @@
     IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
 from orddicts import DefaultOrderedDict
 
 
 vName = {
-    2210: "sea_water_temperature",
-    2220: "sea_water_salinity",
+    #          var name,            units
+    2210: ["sea_water_temperature", "???"],
```

Comment:
Can someone please tell me what the units value should be for this variable? Thanks!

Reply:
I thought you left placeholders on purpose to be imported from the input file in future . From looking at the data, it looks like Celsius for temperature and PSU for salinity.

Reply:
Thanks. I'll use "C" and "PSU".

I was just forcing the tests to fail so that would remind us that the units still needed to get fixed.